common/test/TestModule.cs

```diff
@@ -28,9 +28,7 @@ private TestModule(Module module)
         public static async Task<TestModule> GetTestModuleAsync(string deviceNamePrefix, string moduleNamePrefix)
         {
             var log = TestLogging.GetInstance();
-            string prefix = deviceNamePrefix + "Module" + "_";
-
-            TestDevice testDevice = await TestDevice.GetTestDeviceAsync(prefix).ConfigureAwait(false);
+            TestDevice testDevice = await TestDevice.GetTestDeviceAsync(deviceNamePrefix).ConfigureAwait(false);
```

Comment:
Why didn't you want to prefix with module? It is easy to identify with prefix..isn't it?

Reply:
TestDevice creates the device not the module. The module is then attached to this particular device.